src/main/java/mate/academy/springboot/datajpa/mapper/impl/ProductResponseDtoMapper.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.springboot.datajpa.mapper.impl;
+
+import mate.academy.springboot.datajpa.dto.response.ProductResponseDto;
+import mate.academy.springboot.datajpa.mapper.ResponseDtoMapper;
+import mate.academy.springboot.datajpa.model.Product;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ProductResponseDtoMapper implements ResponseDtoMapper<ProductResponseDto, Product> {
```

Comment:
Please use DtoMapper instead of ProductResponseDtoMapper, check the common mistakes.
Do similar changes for CategoryResponseDtoMapper .